src/node-file-trace.ts

```diff
@@ -54,18 +55,24 @@ export class Job {
   public exportsOnly: boolean;
   public paths: Record<string, string>;
   public ignoreFn: (path: string, parent?: string) => boolean;
+  public stat: (path: string) => Promise<Stats | null>;
+  public readFile: (path: string) => Promise<Buffer | string | null>;
+  public readlink: (path: string) => Promise<string | null>;
+  protected internalStat: (path: string) => Promise<Stats | null>;
```

Comment:
Lets change these to `private` and use `_` to prefix the variable name.

Reply:
done